src/fsharp/CompileOptions.fs

```diff
@@ -733,6 +733,7 @@ let testFlag tcConfigB =
 // not shown in fsc.exe help, no warning on use, motiviation is for use from VS
 let vsSpecificFlags (tcConfigB: TcConfigBuilder) = 
   [ CompilerOption("vserrors", tagNone, OptionUnit (fun () -> tcConfigB.errorStyle <- ErrorStyle.VSErrors), None, None);
+    CompilerOption("validate-type-providers", tagNone, OptionUnit (id), None, None);  // preserved for compatibility's sake, no longer has any effect
```

Comment:
add `Some(DeprecatedCommandLineOptionNoDescription("--validate-type-providers", rangeCmdArgs))`  to mark as deprecated? or not :smile: maybe is useless